README.md

```diff
@@ -174,6 +175,23 @@ make undeploy
 ```sh
 make uninstall
 ```
+### Deploy with OLM
+2. Build and push bundle and index images to an image registry
+```sh
+make build-and-push-bundle-images IMG=<some-registry>/ingress-node-firewall-controller:latest DAEMON_IMG=<some-registry>/ingress-node-firewall-daemom:latest IMAGE_TAG_BASE=<some-registry>/ingress-nodefw/ingress-node-firewall
+```
+3. Deploy with OLM
+```sh
+make deploy-with-olm NAMESPACE=openshift-ingress-node-firewall IMAGE_TAG_BASE=<some-registry>/ingress-nodefw/ingress-node-firewall
+```
+4. Undeploy resources from OCP cluster
+```sh
+oc delete ns openshift-ingress-node-firewall
```

Comment:
can't we use `make undeploy` here ?

Reply:
the make deploy/undeploy deploying from config/manifests, while here we deploy with bundle/manifests so we don't have the same objects running that needs to be cleaned up (like the csv)